frontend/src/page/albums.vue

```diff
@@ -346,7 +348,7 @@ export default {
           {value: 'newest', text: this.$gettext('Newest First')},
           {value: 'oldest', text: this.$gettext('Oldest First')},
           {value: 'added', text: this.$gettext('Recently Added')},
-          {value: 'edited', text: this.$gettext('Recently Edited')},
+          ...(typeName === 'album' ? [{value: 'edited', text: this.$gettext('Recently Edited')}] : [])
```

Comment:
We use the computed `context` property elsewhere for this. Would it be possible to use it here as well?
```js
  computed: {
    context: function () {
      if (!this.staticFilter) {
        return "album";
      }

      if (this.staticFilter.type) {
        return this.staticFilter.type;
      }

      return "";
    }
  },
```

Reply:
yes, updated! I was also thinking instead of `:items="context === 'album' ? options.sorting : options.sorting.filter(item => item.value !== 'edited')"` I could do something like `:items="context === 'album' ? options.albumSorting : options.sorting"` and define a second property `albumSorting` which contains all the album-specific sorting properties, since we might want to add/remove more in the future specifically for albums (for example I think @graciousgrey mentioned we might hide one of the sort options "Newest first" or "Recently Added" for manually managed albums as they are the same). but if we only hide recently edited for now I think this implementation is also just fine, let me know if you have a preference 👍